src/crud.js

```diff
@@ -0,0 +1,80 @@
+const formEl = document.querySelector('.input-container');
+const todoInputEl = document.querySelector('.todo-input');
+const listsEl = document.querySelector('.lists');
+
+const renderTodo = () => {
+  const tasks = JSON.parse(localStorage.getItem('todos')) || [];
+  if (!tasks) return;
+
+  const tasksSorted = tasks.slice().sort((a, b) => a.index - b.index);
+  listsEl.innerHTML = '';
+  tasksSorted.forEach((task) => {
+    const markup = `
+      <li class="list list-${task.index}">
+        <span class="list-left">
+          <button class="btn check" data-btn="${task.index}">
+            <span class="empty-check active"><i class="fa-regular fa-square active"></i></span>
+            <span class="checked"><i class="fa-solid fa-check"></i></span>
+          </button>
+          <input type="text" data-desc="${task.index}" class="todo" value="${task.description}"/>
+        </span>
+        <button class="btn remove" data-remove="${task.index}">
+          <i class="fa-regular fa-trash-can"></i>
+        </button>
+      </>
+    `;
+
+    listsEl.insertAdjacentHTML('beforeend', markup);
+  });
+};
+
+formEl.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const tasks = JSON.parse(localStorage.getItem('todos')) || [];
+  const toDo = todoInputEl.value.trim();
+  todoInputEl.value = '';
+  if (!toDo) return;
+
+  const task = {
+    description: toDo,
+    completed: false,
+    index: tasks.length,
+  };
+
+  const tasksMod = [...tasks, task];
+  localStorage.setItem('todos', JSON.stringify(tasksMod));
```

Comment:
[OPTIONAL] To keep your code more organized, It would be nicer to implement functions for adding a new task, editing task descriptions, and deleting a task instead of writing the script directly in the event listeners.

Reply:
Thank you for reviewing my project, I will make the necessary corrections as suggested.